src/routes/return.js

```diff
@@ -4,44 +4,89 @@ import { Alert } from 'react-bootstrap';
 import NFTCardGrid from '../components/NFTCardGrid';
 import LoginService from '../utils/LoginService';
 import {KasuContract, ERC721Contract} from '../utils/abiManager';
+import { FetchMetadata } from "../utils/opensea";
```

Comment:
@forcey heads up, I added the fetching and displaying metadata logic for the return page. I see you were also working on that page with the modal you added.